spk/java-11-openjdk/Makefile

```diff
@@ -0,0 +1,60 @@
+SPK_NAME = java-11-openjdk
+SPK_VERS = 11.0
+SPK_REV = 1
+SPK_ICON = src/$(SPK_NAME).png
+
+DEPENDS = cross/$(SPK_NAME)
+
+MAINTAINER = SynoCommunity
+DESCRIPTION = The OpenJDK 11 runtime environment
+RELOAD_UI = no
+STARTABLE = no
+DISPLAY_NAME = java-11-openjdk
+CHANGELOG = "Initial version. Based on CentOS 7's java-11-openjdk-11.0.12.0.7-0.el7_9.src.rpm"
+
+HOMEPAGE = http://openjdk.java.net/
+LICENSE  = ASL 1.1 and ASL 2.0 and BSD and BSD with advertising and GPL+ and GPLv2 and GPLv2 with exceptions and IJG and LGPLv2+ and MIT and MPLv2.0 and Public Domain and W3C and zlib
```

Comment:
This is the longest/complicated licence string I've seen. It might be a good idea to include the `LICENSE_FILE = $(WORK_DIR)/xxxxx/LICENSE`

Reply:
Lol you are right. Taken straight from https://src.fedoraproject.org/rpms/java-11-openjdk/blob/rawhide/f/java-11-openjdk.spec#_1155
from where the package is based on.

I think I can replace it with
LICENSE contains:
<html><body>
<!--StartFragment--><pre class="syntaxhighlightblock"><code class="lang-rpm-specfile hljs">

# HotSpot code is licensed under GPLv2
--
  | # JDK library code is licensed under GPLv2 with the Classpath exception
  | # The Apache license is used in code taken from Apache projects (primarily xalan & xerces)
  | # DOM levels 2 & 3 and the XML digital signature schemas are licensed under the W3C Software License
  | # The JSR166 concurrency code is in the public domain
  | # The BSD and MIT licenses are used for a number of third-party libraries (see ADDITIONAL_LICENSE_INFO)
  | # The OpenJDK source tree includes:
  | # - JPEG library (IJG), zlib & libpng (zlib), giflib (MIT), harfbuzz (ISC),
  | # - freetype (FTL), jline (BSD) and LCMS (MIT)
  | # - jquery (MIT), jdk.crypto.cryptoki PKCS 11 wrapper (RSA)
  | # - public_suffix_list.dat from publicsuffix.org (MPLv2.0)
  | # The test code includes copies of NSS under the Mozilla Public License v2.0
  | # The PCSClite headers are under a BSD with advertising license
  | # The elliptic curve cryptography (ECC) source code is licensed under the LGPLv2.1 or any later version

</code></pre><!--EndFragment-->
</body>
</html>